grammar.y

```diff
@@ -262,9 +262,9 @@ select_stmt:
     $1.Limit = $3
     $$ = $1
   }
-| base_select compound_op base_select order_by_opt limit_opt
+| base_select compound_op select_stmt
```

Comment:
This is the fix in the grammar. A select can be composed of a `base_select` + `compound_op` + `select_stmt`. 

Substituting `base_select order_by_opt limit_opt` by `select_stmt` gives the recursive nature that allows multiple compound operators to be used.